tests/molecular/molecules/molecule/conftest.py

```diff
@@ -56,6 +63,9 @@
                 num_repeating_units=2,
             ),
         ),
+        stk.BuildingBlock.init_from_rdkit_mol(
+            molecule=dative_molecule,
+        ),
```

Comment:
This is in the wrong place - this fixture does not ge tused by the get_bonds test. You need to add a CaseData to the building_block fixture defined in this file. You will also probably want to move the dative_molecule closer to the building_block fixter in the file.

Also maybe add tests cases for 

`BuildingBlock('dative smiles')`

too

Reply:
`BuildingBlock('dative smiles')` 
will not work because ETKDG will not run on the BuildingBlock initialisation

Reply:
The bond test is also handled in a future PR when changes to to_rdkit_mol is implemented

Reply:
This test actually requires BuildingBlock.to_rdkit_mol so will come in a future PR and is removed for now.